Tests/XCUITests/DownloadsTests.swift

```diff
@@ -146,8 +145,7 @@ class DownloadsTests: BaseTestCase {
         } else {
             // Workaround to close the context menu.
             // XCUITest does not allow me to click the greyed out portion of the app.
-            app.otherElements["ActivityListView"].cells["XCElementSnapshotPrivilegedValuePlaceholder"].firstMatch.tap()
-            app.navigationBars["Add Tag"].buttons["Done"].tap()
```

Comment:
Oh I think this change is done in a PR that landed earlier today. 
Could you please rebase with latest main? Thanks!

Reply:
Yes it was. I will rebase it.